src/modules/client/mixins/configMixin.js

```diff
@@ -40,9 +40,9 @@ export const configMixin = {
         this.tmpInput = '';
       }
     },
-    nbrError (path) {
-      if (get(this.$v, path).required === false) return REQUIRED_LABEL;
-      if (get(this.$v, path).positiveNumber === false || get(this.$v, path).numeric === false) {
+    nbrError (path, validations) {
```

Comment:
tu peux mettre  `nbrError (path, validations = this.$v)` normalement, pour passer $v en argument par défaut et donc ne pas refacto toutes les fois où cette fonction est appellée

Reply:
oui j'y pensais aussi mais Sophie voulait être plus explicite je crois

Reply:
Plus explicite ? Quesaco ?
Parce que c'est un peu étrange de passer, $v à toutes les fonctions lorsque ça ne sert que pour un seul cas.